examples/stable_fluid.py

```diff
@@ -341,8 +360,15 @@ def reset():
         elif e.key == 'd':
             debug = not debug
 
+    frame += 1
+
+    if frame > 4500:
+        break
+
     if not paused:
-        mouse_data = md_gen(gui)
+        mouse_data = md_gen(gui, frame)
```

Comment:
```suggestion
        mouse_data = md_gen(gui, gui.frame)
```

Reply:
Hi, thanks for the convenient hint~

This part is just for reproducing the same phenomena for test. I will comment out and restore to old, interactive way when finished.